src/ui/SelectionTable.js

```diff
@@ -5,7 +5,12 @@ import { useTable, useRowSelect } from 'react-table';
 import { Checkbox } from './Checkbox';
 import { Box, getBoxProps } from './Box';
 import { uniqueId } from 'lodash';
-import { getValueFromTheme } from './theme';
+import { getValueFromTheme } from '../theme';
+import { Inline } from './Inline';
+import { Button, ButtonVariants } from './Button';
+import { Text } from './Text';
+import { useTranslation } from 'react-i18next';
```

Comment:
I don't think it's a good idea to use translations in our components, I think we should be able to pass them down when using, the same way we did with the modals